test/index.test.ts

```diff
@@ -1,31 +1,58 @@
 import { strictEqual, deepStrictEqual } from "assert";
 import { describe, test } from "mocha";
-import { getAccounts, getConnection } from "@tableland/local";
+import { getAccounts, getDatabase, getValidator } from "@tableland/local";
 
 describe("index", function () {
   // Note that we're using the second account here
   const [, signer] = getAccounts();
-  const sdk = getConnection(signer);
+  const db = getDatabase(signer);
+  const validator = getValidator();
 
   test("create", async function () {
-    const { name } = await sdk.create("counter integer", { prefix: "table" });
-    strictEqual(name, "table_31337_2");
+    const { meta } = await db
+      .prepare("CREATE TABLE my_table (counter integer);")
+      .run();
+    strictEqual(meta.txn?.name, "my_table_31337_2");
   });
 
   test("insert", async function () {
-    const { hash } = await sdk.write("insert into table_31337_2 values (1);");
-    const txnReceipt = await sdk.receipt(hash);
+    const { meta } = await db
+      .prepare("insert into my_table_31337_2 values (1);")
+      .run();
+
+    if (typeof meta.txn?.transactionHash !== "string") {
+      throw new Error("something unexpected happened during insert");
+    }
+    if (typeof meta.txn?.chainId !== "number") {
+      throw new Error("something unexpected happened during insert");
+    }
+
+    const txnReceipt = await validator.receiptByTransactionHash({
+      transactionHash: meta.txn?.transactionHash,
+      chainId: meta.txn?.chainId,
+    });
     strictEqual(txnReceipt?.chainId, 31337);
   });
 
   test("update", async function () {
-    const { hash } = await sdk.write("update table_31337_2 set counter=2;");
-    const txnReceipt = await sdk.receipt(hash);
+    const { meta } = await db
+      .prepare("update my_table_31337_2 set counter=2;")
+      .run();
+
+    strictEqual(typeof meta.txn?.transactionHash, "string");
+    strictEqual(typeof meta.txn?.chainId, "number");
+
+    const txnReceipt = await validator.receiptByTransactionHash({
+      transactionHash: meta.txn?.transactionHash as string,
```

Comment:
We have to use `as` here in order for typescript to build, even though this will never be executed if transactionHash is undefined because of the `strictEqual` check above.

Reply:
Well we can use as here. But you can also use assertions in tests because we _know_ it is going to be valid. Something like this could work:

```suggestion
      transactionHash: meta.txn?.transactionHash!,
```

Reply:
Awesome! this is much nicer to look at and we still get to keep our assertions above. I'll update the other test

Reply:
Spoke too soon here.  This is not allowed by our linter because of @typescript-eslint/no-non-null-assertion

Reply:
I added an `eslint-disable` for the failing rule.  Maybe that combined with the `!` assertion is the best option?